content/en/docs/community/release_checklist.md

```diff
@@ -91,6 +116,13 @@ git checkout -b $RELEASE_BRANCH_NAME
 This new branch is going to be the base for the release, which we are going to
 iterate upon later.
 
+Verify that a [helm/helm milestone](https://github.com/helm/helm/milestones)
+for the release exists on GitHub (creating it if necessary). Make sure PRs and
+issues for this release are in this milestone.
```

Comment:
While the initial setup of the milestone is out of scope here, I think we might want to check it before proceeding.

Reply:
What would be the steps necessary after creating the milestone? If we're this far into the release process and there's no milestone, shouldn't this be a call back to the core maintainers to discuss?

How will this apply to security patches?

Reply:
Do we have a step of making sure that what's in the milestone is also what's being released? Does it not matter at this point since the release notes will be accurate? I'm okay with removing this text if such a check has no function. The function I was imagining is "oh look, we decided to merge this last PR but it's not tagged with the milestone and it's going to be in the release".

Reply:
@bacongobbler This is one of the few remaining questions in my mind before we can merge this PR.

Should we recommend a step of syncing the milestone to the release contents, or is that a step we wouldn't want to suggest?